core/blockchain/blockchain_utils.py

```diff
@@ -72,16 +72,6 @@ def filter_diffs(global_state_wrapper: object, local_state: list,
                             local_state)
     return list(filter(filter_method, new_state))
 
-def update_diffs(global_state_wrapper: object, local_state: list,
```

Comment:
Add the map functionality to `filter_diffs` but have the default be a return present value

Reply:
Done